src/App/components/MediaObject/index.js

```diff
@@ -30,7 +30,9 @@ const MediaObject = ({ size, imgUrl, icon, heading, text, mediaRight }) => {
                 : null}
             <div className="media-body">
                 {mediaHeading()}
-                <p>{text}</p>
+                <p>{textsm ?
```

Comment:
What if I wanna use both text and textsm?
This might make it clearer
```suggestion
               // PropType: textSmall: PropTypes.bool,
                <p>{textSmall ? <small>{text}</small> : text}</p>
```